draft-ietf-taps-transport-security.md

```diff
@@ -844,6 +855,70 @@ and the per-message nonce in the clear. Everything else is encrypted.
 
 - An unreliable transport protocol such as UDP.
 
+## OpenVPN
+
+OpenVPN {{OpenVPN}} is a commonly used protocol designed to replace IPsec. A
+major goal of this protocol is to provide an easy-to-use VPN. OpenVPN
+encapsulates either IP packets or Ethernet frames within a secure tunnel and
+can run over UDP or TCP.
+
+### Protocol Description
+
+OpenVPN facilitates authentication using either a pre-shared static key or
+using X.509 certificates and TLS. In pre-shared key mode, OpenVPN derives
+keys for encryption and authentication directly from one or multiple static
+keys. In TLS mode, OpenVPN encapsulates a TLS handshake, in which both peers
+must present a certificate for authentication. After the handshake, both sides
+contribute random source material to derive keys for encryption and
+authentication using the TLS pseudo random function (PRF). OpenVPN provides the
+possibility to authenticate and encrypt the TLS handshake itself using a
+pre-shared key or passphrase. Furthermore, it supports rekeying using TLS.
+
+After authentication and key exchange, OpenVPN encrypts payload data, i.e., IP
+packets or Ethernet frames, and signs the payload using an HMAC function. The
+default cipher is BlowFish and the default message digest algorithm is SHA1,
+but an application can select an arbitrary cipher, key size, and message digest
+algorithm for the HMAC. OpenVPN peers may also support cipher negotiation
+(NCP). If both hosts allow, they can upgrade the cipher to AES-256-GCM.
+Blocks are padded to an even multiple of block size.
+
+OpenVPN can run over TCP or UDP. When running over UDP, OpenVPN provides a
+simple reliability layer for control packets such as the TLS handshake and key
+exchange. It assigns sequence numbers to packets, acknowledges packets it
+receives, and retransmits packets it deems lost. This reliability layer is not
+used for data packets, which prevents the problem of two reliability mechanisms
+being encapsulated within each other. When running over TCP, OpenVPN includes
+the packet length in the header, which allows the peer to deframe the TCP
+stream into messages.
+
+For replay protection, OpenVPN assigns an identifier to each outgoing packet,
+which is unique for the packet and the currently used key. In pre-shared key
+mode or with a CFB or OFB mode cipher, OpenVPN combines a timestamp with an
+incrementing sequence number into a 64-bit identifier. In TLS mode with CBC
+cipher mode, OpenVPN omits the timestamp, so identifiers are only 32-bit. This
+is sufficient since OpenVPN can guarantee the uniqueness of this identifier for
+each key, as it can trigger rekeying if needed.
+
+OpenVPN supports connection mobility by allowing a peer to change its IP
+address during an ongoing session. When configured accordingly, a host will
+accept packets for a session from any IP address, as long as the packets are
+correctly authenticated.
```

Comment:
What does "correctly authenticated" mean?

Reply:
It means that a packet passes all authentication tests, i.e., it contains a valid HMAC signature.

Maybe "as long as the packets pass all authentication checks" is better? Otherwise, do you have a suggestion?

Reply:
Perhaps: "When configured according, a host will accept authenticated packets for a session from any IP address."

Reply:
Yes, that works much better.